Makefile.js

```diff
@@ -521,12 +522,10 @@ target.gensite = function(prereleaseVersion) {
 
     var docFiles = [
         "/rules/",
-        "/user-guide/command-line-interface.md",
-        "/user-guide/configuring.md",
-        "/developer-guide/code-path-analysis.md",
-        "/developer-guide/nodejs-api.md",
-        "/developer-guide/working-with-plugins.md",
-        "/developer-guide/working-with-rules.md"
+        "/user-guide/",
```

Comment:
@nzakas Now that all the `docFiles` are directory paths, `if (test("-f", fullPath)) { ... }` never executes in step 2 below.

Can you confirm whether that logic has become obsolete, or needs to be rewritten?

If obsolete, I can just delete it when I submit pull request to automatically add rule meta data to docs.